test/unit/components/views/TransactionPage/mocks.js

```diff
@@ -0,0 +1,2009 @@
+const bufferToHex = (dict) => {
+  if (dict === null) {
+    return null;
+  }
+  const arr = [];
+  for (const key in Object.keys(dict)) {
+    arr.push(dict[key]);
+  }
+  return Buffer.from(arr);
+};
+
+const mockRegularTransactionList = [
+  // regular pending transaction
+  {
+    timestamp: 1624527872,
+    height: -1,
+    blockHash: null,
+    index: 0,
+    txHash: "263f64a32f2f86ffda747242cfc620b0c42689f5c600ef2be22351f53bcd5b0d",
+    txType: "regular",
+    debitsAmount: 1793854340,
+    creditsAmount: 993851810,
+    type: 0,
+    amount: -800002530,
+    fee: 2530,
+    debitAccounts: [0],
+    creditAddresses: ["TsacvMFSMWcmxT7dj5UHqgrxB3PP6uwnEtY"],
+    isStake: false,
+    credits: [
+      {
+        index: 0,
+        account: 0,
+        internal: true,
+        amount: 993851810,
+        address: "TsacvMFSMWcmxT7dj5UHqgrxB3PP6uwnEtY",
+        outputScript: "dqkUaU2Rg1LeTP5L7md2o/JQKn/NwviIrA=="
+      }
+    ],
+    debits: [
+      {
+        index: 0,
+        previousAccount: 0,
+        previousAmount: 1793854340
+      }
+    ],
+    rawTx:
+      "010000000137bdd42d19c66cb0ee0da40f5419b284bcd16af1d3836f313fad124b5e5a4fe80000000000ffffffff02a2f93c3b0000000000001976a914694d918352de4cfe4bee6776a3f2502a7fcdc2f888ac0008af2f0000000000001976a9145aec0ddf49a9ca0ad1de2b4143666f544860043688ac000000000000000001840bec6a0000000000000000ffffffff6b4830450221009ce94aac7683dfbac6e23716af87b7529b3cf53d69d1bae7772a9e1baccc873902204e0312be8be7a518723c0c7d9e1f40abe67c681e9aabf78deb21d964c31872c201210314012e939d6ab30c7dacb4258654b0996ce89f7e47faa57bbbf65c44dffd668f",
+    direction: "sent",
+    outputs: [
+      {
+        address: "TsacvMFSMWcmxT7dj5UHqgrxB3PP6uwnEtY",
+        value: 993851810,
+        isChange: true
+      },
+      {
+        address: "TsZJt5A55AcCMp8iBu1rkNCxqJ3Bf1MC8Zk",
+        value: 800000000,
+        isChange: false
+      }
+    ]
+  },
+  // regular received mined regular transaction
+  {
+    timestamp: 1623923254,
+    height: 706945,
+    blockHash: {
```

Comment:
I suggest using a small function to generate this based on the hex string of the block hash you want to use. Defining like this makes this completely unreadable.